supplementary_style_guide/glossary_terms_conventions/general_conventions/h.adoc

```diff
@@ -225,6 +225,17 @@ Use "host" to refer to hosts in general, not "hypervisor", "hypervisor host", or
 
 *See also*:
 
+[discrete]
+[[horizontal-pod-autoscaler]]
+==== image:images/yes.png[yes] horizontal pod autoscaler (noun)
+*Description*: In Red Hat OpenShift, a horizontal pod autoscaler also known as HPA , is implemented as a Kubernetes API resource and a controller. You can use the HPA to specify the minimum and maximum number of pods that you want to run. You can also specify the CPU or memory utilization that your pods should target. The HPA scales out and scales in pods when a given CPU or memory threshold is crossed.
```

Comment:
Suggest adding a comma and removing a space

```suggestion
*Description*: In Red Hat OpenShift, a horizontal pod autoscaler, also known as HPA, is implemented as a Kubernetes API resource and a controller. You can use the HPA to specify the minimum and maximum number of pods that you want to run. You can also specify the CPU or memory utilization that your pods should target. The HPA scales out and scales in pods when a given CPU or memory threshold is crossed.
```

Reply:
I'd also suggest changing 'utilization' to 'usage' (because IBM say to avoid 'utilization').
`You can also specify the CPU or memory usage that your pods target.`

And this sentence reads a little strangely to me:
`The HPA scales out and scales in pods when a given CPU or memory threshold is crossed.`
Maybe it could be simplified something like this:
`The HPA scales in or out when a given CPI or memory threshold is crossed.`

Reply:
Re-phrased to add **Pods** thanks